NN_start.js

```diff
@@ -53,6 +54,33 @@ NerthusAddonUtils = (function()
     }
     utils.runAddon = function()
     {
+        nerthus.interface = (function()
```

Comment:
to bym zrobił inaczej, mam już pomysł, dorzucę to potem

Reply:
Można ewentualnie sprawdzać która zmienna jest zdefiniowana (g czy API), ale za pomocą ciasteczka sprawdza to sam silnik gry, więc myślę że tak jest "bezpiecznej"

Reply:
miałem na myśli raczej całe uruchamianie dodatku NI/SI niż tę linijkę

Reply:
To pytałem o to wcześniej w issue .-. no nieważne, jak dorzucisz to ok.

Reply:
Zmiany w NN_start.js powinny już być niepotrzebne jak zsynchronizujesz repo z moim masterem